index.js

```diff
@@ -157,7 +158,7 @@ function compression(options) {
 
       // compression method
       var accept = accepts(req)
-      var method = accept.encoding(['gzip', 'deflate', 'identity'])
+      var method = accept.encoding(available)
 
       // negotiation failed
       if (!method || method === 'identity') {
```

Comment:
before force `identity`, people should check the presence of their encoding inside this module

worst case? `ciao` like a new superpowerful algorithm :)
`['ciao', 'identity']` as option
`ab -v 4 -H "Accept-Encoding: ciao" -c 1 -n 1 127.0.0.1:3000/with_compression`
```shell
LOG: header received:
HTTP/1.1 200 OK
Content-Type: text/plain
Vary: Accept-Encoding
Content-Encoding: ciao
Date: Mon, 26 Jan 2015 18:59:20 GMT
Connection: close
```
sent with `deflate` algorithm https://github.com/expressjs/compression/blob/master/index.js#L172